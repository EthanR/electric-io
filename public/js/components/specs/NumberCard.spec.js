import { shallowMount } from "@vue/test-utils";
import NumberCard from "../NumberCard";

describe("Number card", () => {
  test("component can mount", () => {
    const wrapper = shallowMount(NumberCard);
    expect(wrapper.isVueInstance()).toBeTruthy();
  });

  test("renders with color and number value", () => {
    const wrapper = shallowMount(NumberCard, {
      propsData: {
        tile: {
          textColor: "blue"
        }
      },
      data: () => ({
        number: 1
      })
    });

    expect(wrapper.html()).toMatchSnapshot();
  });
});
