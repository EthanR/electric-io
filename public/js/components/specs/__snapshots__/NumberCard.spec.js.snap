// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Number card renders with color and number value 1`] = `"<p class=\\"number\\" style=\\"color: blue;\\">1</p>"`;
